lib/stringify.js

```diff
@@ -112,6 +119,12 @@ module.exports = function (object, opts) {
     var sort = typeof options.sort === 'function' ? options.sort : null;
     var allowDots = typeof options.allowDots === 'undefined' ? false : options.allowDots;
     var serializeDate = typeof options.serializeDate === 'function' ? options.serializeDate : defaults.serializeDate;
+    if (typeof options.format === 'undefined') {
+        options.format = defaults.format;
+    } else if (!(options.format in defaults.formatters)) {
```

Comment:
this will allow a format of `toString`, for example - instead of `in`, could we use `Object.prototype.hasOwnProperty.call`?